index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img src="images/mul21.png" alt="mul21">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading">Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      <section class="ab">
+        <div class="about">
+  <div class="about__content">
+    <h2 class="aheading">About Myself</h2>
+    <p class="ades">
+      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p class="aget">LET'S CONNECT</p>
+    <ul class="socialmediaicons" >
+      <li>
+        <a class="social" href="https://twitter.com/hotumanzade">
+          <img src="Images\twitter.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a class="social" href="https://www.linkedin.com/in/burak-otman-88646443/">
+          <img src="Images\Linkedin icon.png" alt="link"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="#">
+          <img src="Images\m.png" alt="Angelist"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="https://github.com/otmanTR">
+          <img src="Images\kedi.png" alt="github"> </a>
+        </a>
+      </li>
+      <li>
+        <a class="social" href="#">
+          <img src="Images\vin.png" alt="medium"> </a>
+        </a>
+      </li>
+   </ul>
+   <button class="btn">Get my resume</button>
+  </div>
+  <div class="About__skills">
+    <ul class="lan"><li class="lan1">Languages <img class="disable3" src="/Images/Vektor.png"></li>
```

Comment:
- [ ] Please consider removing the `/` as mentioned by the previous reviewer 🤝🏽. If you want to specify the relative path you can use `./` ✔ but not only `/` ❌. That way when displaying the images it will check from the root directory.